recipe/meta.yaml

```diff
@@ -12,33 +12,43 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
+  skip: true  # [py<37]
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
+    - wheel
     - python
-    - setuptools
-    - setuptools_scm
+    - setuptools >=42
+    - setuptools_scm >=3.4.1
     - toml
   run:
     - python
-    - importlib_metadata
+    - importlib_metadata  # [py<38]
 
 test:
+  requires:
+    - pip
   imports:
     - jsonpickle
+  commands:
+    - pip check
 
 about:
-  summary: Python library for serializing any arbitrary object graph into JSON. It can take almost any Python object and turn the object into JSON. Additionally,
-    it can reconstitute the object back into Python.
   home: http://jsonpickle.github.io/
```

Comment:
Could you please fix home url with HTTPS?

Reply:
yes, will do!